### PR TITLE
Implement account details modal (with rename account feature)

### LINKED ISSUE #589: Wallets Page Improvements (Account Properties view)
Display account properties when user taps on an account from the wallets/accounts table view.

**Initial view**:
https://gallery.io/projects/MCHbtQVoQ2HCZZTQVworZ4Kv/files/MCEJu8Y2hyDScSnu1sfWNvvvMAzwq17b6C0

**Initial view (with other balances)**:
https://gallery.io/projects/MCHbtQVoQ2HCZZTQVworZ4Kv/files/MCEJu8Y2hyDScTokzG0AlQz5HZ_Lb3wBP-g

**On view property toggle:**:
https://gallery.io/projects/MCHbtQVoQ2HCZZTQVworZ4Kv/files/MCEJu8Y2hyDScQRfygISNs0r31I78sNS44I

### PR DESCRIPTION
**Pull request by itswisdomagain**
Resolves #589. Requires https://github.com/raedahgroup/dcrlibwallet/pull/98.

### CODE REVIEW

File: Decred Wallet/Features/Wallets/Wallets.storyboard [hunk 1]
```diff
@@ -401,12 +401,396 @@
...
+                                        <real key="value" value="14"/>
+                                    </userDefinedRuntimeAttribute>
+                                </userDefinedRuntimeAttributes>
+                            </view>
+                        </subviews>
+                        <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                        <constraints>
+                            <constraint firstItem="ggu-IF-Ehw" firstAttribute="leading" secondItem="ArT-Pb-cdS" secondAttribute="leading" id="1mp-JX-0HT"/>
+                            <constraint firstItem="ggu-IF-Ehw" firstAttribute="trailing" secondItem="ArT-Pb-cdS" secondAttribute="trailing" id="VKB-o2-RFR"/>
+                            <constraint firstItem="ArT-Pb-cdS" firstAttribute="bottom" secondItem="ggu-IF-Ehw" secondAttribute="bottom" id="mHX-Tv-9CH"/>
```
Comment (bgptr) [inline Decred Wallet/Features/Wallets/Wallets.storyboard]: The `secondItem` should be the `SafeArea` and the `StackView`'s `secondItem` should be the `superView`. See the suggestion below.
```suggestion
                            <constraint firstAttribute="bottom" secondItem="ViB-dv-2vV" secondAttribute="bottom" id="z8U-kM-Uhk"/>
```

Comment (itswisdomagain) [reply Decred Wallet/Features/Wallets/Wallets.storyboard]: Funnily, that is actually relative to `Safe Area.bottom`. Not sure why that issue is there but will check.

Comment (itswisdomagain) [reply Decred Wallet/Features/Wallets/Wallets.storyboard]: Ought to have been constraint to `Superview.bottom`. Fixed in https://github.com/raedahgroup/dcrios/pull/592/commits/8a5ceee3149eab96a16f24e38db46b4e80af8c65.

File: Decred Wallet/Features/Wallets/Wallets.storyboard [hunk 2]
```diff
@@ -762,7 +762,7 @@
                         <constraints>
                             <constraint firstItem="ggu-IF-Ehw" firstAttribute="leading" secondItem="ArT-Pb-cdS" secondAttribute="leading" id="1mp-JX-0HT"/>
                             <constraint firstItem="ggu-IF-Ehw" firstAttribute="trailing" secondItem="ArT-Pb-cdS" secondAttribute="trailing" id="VKB-o2-RFR"/>
-                            <constraint firstItem="ArT-Pb-cdS" firstAttribute="bottom" secondItem="ggu-IF-Ehw" secondAttribute="bottom" id="mHX-Tv-9CH"/>
+                            <constraint firstAttribute="bottom" secondItem="ggu-IF-Ehw" secondAttribute="bottom" id="mHX-Tv-9CH"/>
```
Comment (bgptr) [inline Decred Wallet/Features/Wallets/Wallets.storyboard]: I rather recommend setting the bottom constraint of `stackView` to `safeArea`. 
Delete line 748 and put this line after line 765:

```<constraint firstItem="ViB-dv-2vV" firstAttribute="bottom" secondItem="ArT-Pb-cdS" secondAttribute="bottom" id="nwZ-Jc-YUw"/>```

Comment (itswisdomagain) [reply Decred Wallet/Features/Wallets/Wallets.storyboard]: Perfect! I was wondering how to achieve that perfect bottom constraint for different device configs. This is it.

**Pull request merged**